packages/ses/index.test-d.ts

```diff
@@ -206,4 +206,4 @@ expectType<Error>(assert.error(X`details are ${stringable}`, TypeError));
 
 expectType<Error>(assert.error(X`details are ${stringable}`, TypeError, { errorName: 'Nom de plum' }));
 
-expectType<never>(assert.fail(X`details are ${stringable}`));
```

Comment:
we shouldn't lose this test about `assert.fail`

Reply:
If I include both, lint says the second is unreachable. Since, IIUC, the point of this test is to statically test what typescript says and the unreachability error is lint, not ts, I'm suppressing the lint error. But I'm not really sure I understand `expectType` enough, so I'm leaving this conversation unresolved until you PTAL. Thanks.

Reply:
That's fine. Another option would be to put it into a function body.